src/Icons/doc/index.rst

```diff
@@ -124,8 +299,55 @@ This command looks in all your twig templates for ``ux_icon`` calls and caches t
 
     If using `symfony/asset-mapper`_, the cache is warmed automatically when running ``asset-map:compile``.
 
-Full Default Configuration
---------------------------
+TwigComponent
+~~~~~~~~~~~~~
+
+The ``ux_icon`` function is optimized to be as fast as possible. To deliver the same level
+of performance with the TwigComponent (``<twig:UX:Icon name="..." />``), the TwigComponent
+usual overhead is reduced to the bare minimum, immediately calling the IconRenderer and
+returning the HTML output.
+
+.. warning::
+
+    The <twig:UX:Icon> component does not support embedded content.
+
+    .. code-block:: twig+html
+
+        {# The 🧸 will be ignore in the HTML output #}
+        <twig:UX:Icon name="user-profile" class="w-4 h-4">🧸</twig:UX:Icon>
+
+        {# Renders "user-profile.svg" #}
+        <svg viewBox="0 0 24 24" class="w-4 h-4">
+            <path fill="currentColor" d="M21 7L9 19l-5.5-5.5l1.41-1.41L9 16.17L19.59 5.59z"/>
+        </svg>
+
+
+Configuration
```

Comment:
I think we don't need this section because we already show that in other part of this doc.